fpgaconvnet/models/layers/MultiPortLayer.py

```diff
@@ -108,8 +108,11 @@ def cols(self, val: List[int]) -> None:
 
     @channels.setter
     def channels(self, val: List[int]) -> None:
-        assert(len(val) == self.ports_in)
-        self._channels = val
```

Comment:
Lets keep this instead, as it explicit that it takes a list of channels as the input